src/tabs/tabs.js

```diff
@@ -6,6 +6,10 @@ angular.module('ui.bootstrap.tabs', [])
   ctrl.tabs = [];
 
   ctrl.select = function(index, evt) {
+    if (evt && evt.which === 2) {
```

Comment:
I'm confused.  According to this: https://developer.mozilla.org/en-US/docs/Web/Events/click, the middle button is 1.